bin/start.ts

```diff
@@ -1,14 +1,16 @@
-import { fork, receive } from 'effection';
+import { main, fork, receive, Operation } from 'effection';
 import * as tempy from 'tempy';
 
 import { createOrchestrator } from '../src/index';
 
-fork(function*() {
-  let interrupt = () => { this.halt()};
-  process.on('SIGINT', interrupt);
+const self: Operation = ({ resume, context: { parent }}) => resume(parent);
 
+main(function*() {
+  let context = yield self;
+  let interrupt = () => { context.halt()};
+  process.on('SIGINT', interrupt);
   try {
-    fork(createOrchestrator({
+    let orchestrator = yield fork(createOrchestrator({
```

Comment:
In order to "execute" the fork operation, you need to yield to it. The result of the operation is the asynchronous child context. This will return immediately.